qiskit/qasm3/experimental.py

```diff
@@ -68,3 +68,7 @@ class ExperimentalFeatures(enum.Flag):
             }
         }
     """
+    GLOBAL_BARRIER = enum.auto()
+    """Support exporting using golobal barrier as proposed in
+        https://github.com/Qiskit/qiskit/issues/13485
+    """
```

Comment:
It'd be a little cleaner to just explain inline what's meant, rather than linking to a Qiskit issue.  The "global barrier" is already a well-defined feature of OpenQASM 3, so the documentation might more appropriately note why it's not the default behaviour (not every consumer of Qiskit's OQ3 output supports it), and say something like

> Export `` :class:`.Barrier` `` objects that act on all qubit circuit qubits as global ``barrier;`` statements, rather than explicitly listing qubits.

It's a bit easier for a user to follow, without needing to click links and read technical discussions.